### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Rafiduzzaman**
## In this part the contact form is modified to get user data and send them through the contact form to Formspree.
-Implement contact form 
For Respected reviewers,
the webhint linters didn't pass because of an interior problem and are supposed to be excused as per the announcement by the Microverse team. Kindly check on the factor. Thanks sincerely.

### CODE REVIEW

**Review by laiifuu — COMMENTED** [review]
# STATUS: INVALID :stop_sign: 

## Hi team :wave: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project where the linter result is red. For your project to be reviewed, linters check must be green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -254,16 +259,28 @@ <h1 id="abt_mslf">About Me</h1>
...
   <!-- Contact  -->
   <section class="contact">
     <h1 id="headerrr">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h1>
-    <input type="text" placeholder="Full Name" id="name">
-    <input type="text" placeholder="Email Address" id="email">
-    <input type="text" placeholder="Write me something..." id="para">
-    <button id="button1">Get in touch</button>
-    <hr>
+    <form action="https://formspree.io/f/mqkorarr" method="post">
```
Comment (brainconnect93) [inline index.html:268]: - [ ] Great effort working on the contact section of the portfolio project, but currently, i noticed that i could not submit the form after i have inserted all the necessary data. I would kindly suggest you work on the `formspree` and be sure the id is properly used so as to aid submission of data inserted by a user.

Video claim 👇


https://github.com/Rdzaman20000/Portfolio/assets/78634841/4ff0c6cc-63c6-4dff-9570-4874ea6f0df5

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Rdzaman20000 , 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the required attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Pair-programming collaborative activity ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Rdzaman20000) [reply index.html:266]: @brainconnect93 I have implemented the button for the mobile version and not the desktop one yet. so kindly try to submit it using the mobile version e.g reducing .  Thanks

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Rdzaman20000,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**